jsconfig.json

```diff
@@ -24,7 +24,16 @@
          * Typecheck JS in `.svelte` and `.js` files by default.
          * Disable this if you'd like to use dynamic types.
          */
-        "checkJs": true
+        "checkJs": true,
+        "paths": {
+            "$src/*": ["src/*"],
+            "$base_components/*": ["src/base_components/*"],
+            "$assets/*": ["src/assets/*"],
+            "$lib/*": ["src/lib/*"],
+            "$navigation/*": ["src/lib/navigation/*"],
```

Comment:
I am not sure about this... We will potentially add lots of these in projects and then we always have to come and update this...

Reply:
Happy to have the $src one...